Increase the send queue limit to avoid metrics (#383)
infrawatch/documentation: common/collectd/ref_collectd-plugins.adoc

|Integer
|===

> The `send_queue_limit` parameter is used to limit the length of the queue to store outgoing metrics. If there is no

Reviewer: ```suggestion
Use the `send_queue_limit` parameter to limit the length of the outgoing metrics queue. If there is no
```

Reviewer: Is it possible to add a third column to Table 2. amqp1 parameters for descriptions? If so, you could add the description of the `send_queue_limit` parameter to it.

> AMQP1 connection, the plugin will continue to queue messages to send, which can result in unbounded memory consumption.

Reviewer: ```suggestion
[NOTE] If there is no AMQP1 connection, the plugin continues to queue messages to send, which can result in unbounded memory consumption.
```

Also suggest moving this note to the end of the description.

> The default value is 0, which disables this feature. If metrics are missing, it may make sense to increase the parameter.

Reviewer 2: Is there any logging or metrics that can be viewed to know when this upper bound is being hit?

Author: There is no logging.

> The default value is 0, which disables this feature. If metrics are missing, it may make sense to increase the parameter.

Reviewer: ```suggestion
The default value is 0, which disables the outgoing metrics queue. 

Increase the value of the `send_queue_limit` parameter if metrics are missing.
```
Do you use a TIP admonition? If so, you could make the "Increase the value ..." a tip.

|Integer
|===

> Use the `send_queue_limit` parameter to limit the length of the outgoing metrics queue. If there is no

Reviewer: ```suggestion
Use the `send_queue_limit` parameter to limit the length of the outgoing metrics queue.
```
Removing a duplicate

> [NOTE] If there is no AMQP1 connection, the plugin continues to queue messages to send, which can result in unbounded memory consumption.

Reviewer 2: Have you run `make` build? I'm not sure if this will result in the proper note callout if there isn't a newline separating line 207 and 208.

Author: fixed.
